UT4MasterServer.Services/Scoped/AwsSesClient.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Amazon.SimpleEmail;
+using Amazon.SimpleEmail.Model;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Text.Json;
+using UT4MasterServer.Common.Exceptions;
+using UT4MasterServer.Models.Settings;
+using UT4MasterServer.Services.Interfaces;
+
+namespace UT4MasterServer.Services.Scoped;
+
+public sealed class AwsSesClient : IEmailService
+{
+	private readonly ILogger<AwsSesClient> _logger;
+	private readonly IAmazonSimpleEmailService _amazonSimpleEmailService;
+
+	public AwsSesClient(ILogger<AwsSesClient> logger, IOptions<AWSSettings> awsSettings)
+	{
+		_logger = logger;
+		_amazonSimpleEmailService = new AmazonSimpleEmailServiceClient(
+			awsSettings.Value.AccessKey,
+			awsSettings.Value.SecretKey,
+			Amazon.RegionEndpoint.GetBySystemName(awsSettings.Value.RegionName));
```

Comment:
I didn't check what happens, but server shouldn't crash if these are not correct. Instead errors should be logged.

Reply:
I don't think that server will crash, instead global error handler will catch the error.